doc/modules/manifold.rst

```diff
@@ -479,12 +479,37 @@ t-distributed Stochastic Neighbor Embedding (t-SNE)
 t-SNE (:class:`TSNE`) converts affinities of data points to probabilities.
 The affinities in the original space are represented by Gaussian joint
 probabilities and the affinities in the embedded space are represented by
-Student's t-distributions. The Kullback-Leibler (KL) divergence of the joint
+Student's t-distributions. This allows t-SNE to be particularly sensitive
+to local structure and has a few other advantages over existing techniques:
+
+* Revealing the structure at many scales on a single map
+* Revealing data that lie in multiple, different, manifolds or clusters
+* Reducing the tendency to crowd points together at the center
+
+While Isomap, LLE and variants are best suited to unfold a single continuous
```

Comment:
I didn't check but that sounds quite verbatim from the paper. If it is, we should probably make that explicit.